libsolidity/ast/Types.h

```diff
@@ -1528,6 +1528,8 @@ class MappingType: public CompositeType
 	bool hasSimpleZeroValueInMemory() const override { solAssert(false, ""); }
 	bool nameable() const override { return true; }
 
+	std::vector<std::tuple<std::string, Type const*>> makeStackItems() const override;
```

Comment:
Where is `makeStackItems()` used?

Reply:
It's used in Types.h is one single usage :D